Tests/TemplatesTests/JsonTests.swift

```diff
@@ -9,12 +9,12 @@ import XCTest
 
 final class JsonTests: XCTestCase {
   private enum Contexts {
-    static let all = ["empty", "all"]
+    static let all = ["empty", "array", "configuration"]
```

Comment:
This should be changed back once we have the `all` context again (see comment above).